File: Day00/part01.py
import os

file = os.path.join(os.path.dirname(__file__), "blob.txt")


def file_reader(file: str) -> str:
    with open(file, "r") as f:
        return f.read()


def solver(blob: str) -> int:
    for b in blob.splitlines():
        ...
    return 0


if __name__ == "__main__":
    print(solver(file_reader(file)))

File: Day00/part01.go
package main

import (
	"bufio"
	"fmt"
	"log"
	"os"
)

func fileReader(file string) []string {

	f, err := os.Open(file)
	if err != nil {
		log.Fatal(err)
	}
	defer f.Close()
	var ret []string
	scanner := bufio.NewScanner(f)

	for scanner.Scan() {
		ret = append(ret, scanner.Text())
	}

	return ret
}

func solver(s []string) int64 {
	var seats []int64

	for _, s := range s {
	}

	return 0
}

func main() {
	fmt.Println(solver(fileReader("./blob.txt")))
}
